go.mod

```diff
@@ -3,23 +3,22 @@ module github.com/elastic/beats/v7
 go 1.16
 
 require (
-	cloud.google.com/go v0.51.0
-	cloud.google.com/go/bigquery v1.0.1
-	cloud.google.com/go/pubsub v1.0.1
-	cloud.google.com/go/storage v1.0.0
+	cloud.google.com/go v0.83.0
+	cloud.google.com/go/bigquery v1.8.0
+	cloud.google.com/go/pubsub v1.3.1
+	cloud.google.com/go/storage v1.10.0
 	code.cloudfoundry.org/go-diodes v0.0.0-20190809170250-f77fb823c7ee // indirect
 	code.cloudfoundry.org/go-loggregator v7.4.0+incompatible
 	code.cloudfoundry.org/rfc5424 v0.0.0-20180905210152-236a6d29298a // indirect
 	github.com/Azure/azure-event-hubs-go/v3 v3.1.2
-	github.com/Azure/azure-sdk-for-go v37.1.0+incompatible
+	github.com/Azure/azure-sdk-for-go v55.2.0+incompatible
```

Comment:
It's not that simple to upgrade this library since the interfaces have been changed and requires some tuning. Fortunately it will be upgraded properly by https://github.com/elastic/beats/pull/28028 so we can wait for it.